addons/sale/tests/test_sale_order_discount.py

```diff
@@ -108,6 +108,19 @@ def test_percent_discount_above_100(self):
         with self.assertRaises(ValidationError):
             self.wizard.write({'discount_percentage': 1.1, 'discount_type': 'sol_discount'})
 
+    def test_discount_translation(self):        
```

Comment:
```suggestion
    def test_discount_translation(self):
```
try looking into adding a linter or configuring your IDE to warn you about trailing whitespace